client/src/components/atoms/TextArea.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from "react";
+import styled from "@emotion/styled";
+import { Text } from "@/components/atoms";
+
+export const Container = styled(Text)`
+  width: 100%;
+  outline: none;
+`;
+
+export const TextArea: React.FC = () => {
+  const [message, setMessage] = useState<string>("");
+  return (
+    <Container variant="body" onInput={() => {}} contentEditable>
+      {/* TODO: max char limit, emoticon and hyperlink */}
+      {message}
+    </Container>
+  );
+};
```

Comment:
요런식으로 하면 조금 더 간결할 것 같아요
```suggestion
export const TextArea = styled(Text)`
  width: 100%;
  outline: none;
`;

TextArea.defaultProps = { contentEditable: true };
```